compatibility_spark-3/src/test/resources/delta-cdc-to-snapshot/test-data/first-input.csv

```diff
@@ -0,0 +1,26 @@
+id1,value01,2022-02-05 14:11:30.106,PT
+id2,value02,2022-02-05 14:11:30.106,PT
+id2,value03,2022-02-06 14:11:30.106,UP
+id2,value04,2022-02-07 14:11:30.106,UP
+id3,value05,2022-02-05 14:11:30.106,PT
+id3,value06,2022-02-05 14:11:30.106,UP
+id3,value07,2022-02-05 14:11:30.106,RR
+id4,value08,2022-02-05 14:11:30.106,PT
+id4,value09,2022-02-06 14:11:30.106,UP
+id4,value10,2022-02-06 14:11:30.106,RR
+id5,value11,2022-02-05 14:11:30.106,PT
+id5,value12,2022-02-05 14:11:30.106,UP
+id5,value13,2022-02-06 14:11:30.106,RR
```

Comment:
Isn't the testcase for id5 already covered with id4?

Reply:
Dates are different. id4 = last two are in conflict and id5 = first two